### PR TITLE
Update _codesandbox.html.erb

### PR DESCRIPTION
**Pull request by garethx**
What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Documentation Update

## Description
Adding 'allow-popups' to enable 'open sandbox' and link buttons to work, so folks can open the sandbox in a new tab.

### CODE REVIEW

**Comment by rhymes** [comment]
Hi @garethx, thank you for your PR! 

could you provide a link to the option and/or a video of how the feature works?

Also, I've noticed you linked https://github.com/thepracticaldev/dev.to/issues/4252 but I guess it's not the issue you're referring to, correct?

**Comment by garethx** [comment]
It fixes this:

Screenshot 2019-10-24 at 13 16 02

Yeah, I don't know how that number got there, there's no corresponding issue I'm aware of.

File: app/views/liquids/_codesandbox.html.erb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <iframe
   src="<%= "https://codesandbox.io/embed/#{id}#{query}" %>"
   style="width:100%; height:calc(300px + 8vw); border:0; border-radius: 4px; overflow:hidden;"
-  sandbox="allow-same-origin allow-scripts allow-forms allow-top-navigation-by-user-activation">
+  sandbox="allow-same-origin allow-scripts allow-forms allow-popups allow-top-navigation-by-user-activation">
```
Comment (MichaelDeBoey) [inline app/views/liquids/_codesandbox.html.erb]: @garethx If you follow the manual steps, you'll get

```html
<iframe
  allow="geolocation; microphone; camera; midi; vr; accelerometer; gyroscope; payment; ambient-light-sensor; encrypted-media; usb"
  sandbox="allow-modals allow-forms allow-popups allow-scripts allow-same-origin"
  src="https://codesandbox.io/embed/react-timeline-demo-1-srpe2?fontsize=14"
  style="width:100%; height:500px; border:0; border-radius: 4px; overflow:hidden;"
  title="react-timeline-demo-1"
></iframe>
```

So I think you should make this
```suggestion
  allow="geolocation; microphone; camera; midi; vr; accelerometer; gyroscope; payment; ambient-light-sensor; encrypted-media; usb"
  sandbox="allow-modals allow-forms allow-popups allow-scripts allow-same-origin">
```

Comment (garethx) [reply app/views/liquids/_codesandbox.html.erb]: Thanks!

Comment (MichaelDeBoey) [reply app/views/liquids/_codesandbox.html.erb]: Will create a PR for the docs, so it's in sync there too 🙂

**Comment by benhalpern** [comment]
Would adding `allow-popups` allow users to create popups or is this strictly behavior the Code Sandbox app would trigger?

**Comment by garethx** [comment]
It would allow users to embed sandboxes that potentially create pop-ups as the underlying embed uses an iframe with `allow-popups` too.

**Pull request merged**